packages/frontend/src/fetcher/on-chain/index.ts

```diff
@@ -9,26 +8,30 @@ import {
     REALITY_TEMPLATE_OPTIONS,
     SupportedChainId,
 } from "../../commons";
-import REALITY_ETH_V3_ABI from "../../abis/reality-eth-v3.json";
+import REALITY_ETH_V3_ABI from "../../abis/reality-eth-v3";
 import { enforce, isCID } from "@carrot-kpi/sdk";
 import {
     RealityResponse,
     OnChainRealityQuestion,
     RealityQuestion,
 } from "../../page/types";
-import { BigNumber, utils } from "ethers";
-import { defaultAbiCoder } from "ethers/lib/utils.js";
+import {
+    decodeAbiParameters,
+    getContract,
+    keccak256,
+    parseAbiItem,
+    toHex,
+} from "viem";
+import { type Address } from "viem";
 
-const LOGS_BLOCKS_SIZE = __DEV__ ? 10 : 10_000;
-const NEW_QUESTION_LOG_TOPIC = utils.solidityKeccak256(
-    ["string"],
-    [
-        "LogNewQuestion(bytes32,address,uint256,string,bytes32,address,uint32,uint32,uint256,uint256)",
-    ]
+const LOGS_BLOCKS_SIZE = __DEV__ ? 10n : 10_000n;
+const NEW_QUESTION_LOG_TOPIC = keccak256(
+    toHex(
+        "LogNewQuestion(bytes32,address,uint256,string,bytes32,address,uint32,uint32,uint256,uint256)"
+    )
 );
-const NEW_ANSWER_LOG_TOPIC = utils.solidityKeccak256(
-    ["string"],
-    ["LogNewAnswer(bytes32,bytes32,bytes32,address,uint256,uint256,bool)"]
+const NEW_ANSWER_LOG_TOPIC = keccak256(
+    toHex("LogNewAnswer(bytes32,bytes32,bytes32,address,uint256,uint256,bool)")
```

Comment:
And here too.

Reply:
Fixed in 07174a125f90bbdd3a733f0972d6eab14b0c54e6.